### DISCUSSION TITLE
Auto push commit cannot trigger workflow while manual push can

### DISCUSSION THREAD

**Discussion by tisonkun** [community/community#40798] [Actions] [answered]
### Select Topic Area

Bug

### Body

https://github.com/apache/pulsar-site/commits/main

The workflow file `ci-build-site.yml` has a `push` trigger:

```yml
on:
  push:
    branches: [main]
```

While manual push can trigger the workflow, if the commit is made by automation like https://github.com/apache/pulsar-site/commit/d04cf096f49b60cd689fc6894c06263736db2af4, the workflow isn't triggered.

**Comment by airtower-luna** [comment, marked as answer]
How is the push in automation authorized? If it's in another workflow using the `GITHUB_TOKEN` it [won't trigger new workflows](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow).

**Reply by tisonkun** [reply]
Thank you! This is the case.
